### ISSUE TITLE
Figcaption example overly detailed

### ISSUE THREAD

**Issue by CharlesBelov** [w3c/alt-techniques#2]
I have a nitpick with the section on [The figure and figcaption elements](http://www.w3.org/TR/html-alt-techniques/#hf). where it gives "Shadow like figures and a graffiti tag drawn on the walls of a partially demolished building, illuminated by the light from a street lamp (photo)." both as alt text and as a caption. I would have used the full version as alt text but an edited version "Drawings on the walls of a partially demolished building (photo)." as the caption. Editorially, I feel the full version is too wordy to serve as a good caption. 

That would actually be a good example of where a captioned photo nevertheless also needs alternative text.

A good example of a captioned image that did not need alternative text would be a simple image where a few words would be sufficient as alternative text.

**Comment by liamquin** [comment]
I agree. This example was problematic because the picture element wasn't final, but that's no longer an issue. Accepted, thank you for the suggestion.
